fabric-entity-events-v1/src/main/java/net/fabricmc/fabric/api/entity/event/v1/ServerPlayerEvents.java

```diff
@@ -62,6 +62,17 @@ public final class ServerPlayerEvents {
 		return true;
 	});
 
+	/**
+	 * An event that is called before a player is initially spawned.
```

Comment:
I suggest mentioning that this is only called when the player first connects to the server. Also, I think it's better to put this above `COPY_FROM`.

Reply:
I meant that this event is not invoked on respawn. Maybe direct people to use `COPY_FROM` if they want to be invoked on respawn.

Reply:
Yeah ok, I think its good to have all that info in the docs.